client/my-sites/stats/stats-notices/index.tsx

```diff
@@ -40,10 +45,15 @@ const NewStatsNotices = ( { siteId, isOdysseyStats }: NewStatsNoticesProps ) =>
 	const { hasLoadedPurchases } = usePurchasesToUpdateSiteProducts( isOdysseyStats, siteId );
 
 	const showPaidStatsNotice =
-		config.isEnabled( 'stats/paid-stats' ) &&
-		! isVip &&
-		! isP2 &&
-		! isOwnedByTeam51 &&
+		// Show the notice if the site is Jetpack or it is Odyssey Stats.
+		( ( config.isEnabled( 'stats/paid-stats' ) && ( isOdysseyStats || isSiteJetpackNotAtomic ) ) ||
```

Comment:
IMHP, it would be better to refactor as variables for readability. 🙂

Reply:
agreed will refactor later. Thanks 🙏